config/v1/types_ingress.go

```diff
@@ -54,6 +54,24 @@ type IngressSpec struct {
 	// configurable routes.
 	// +optional
 	ComponentRoutes []ComponentRouteSpec `json:"componentRoutes,omitempty"`
+
+	// requiredHSTSPolicies specifies HSTS policies that are required to be set on
+	// newly created routes matching criteria that are specified in the policy.
```

Comment:
newly create routes and routes that are updating the HSTS policy annotation (can you write out the actual annotation name here?)

Reply:
These comments have been clarified.